backend/src/test/scala/com/softwaremill/bootzooka/passwordreset/PasswordResetApiTest.scala

```diff
@@ -1,22 +1,14 @@
 package com.softwaremill.bootzooka.passwordreset
 
-import cats.effect.IO
 import com.softwaremill.bootzooka.email.sender.DummyEmailSender
 import com.softwaremill.bootzooka.infrastructure.Json._
-import com.softwaremill.bootzooka.passwordreset.PasswordResetApi.{
-  ForgotPassword_IN,
-  ForgotPassword_OUT,
-  PasswordReset_IN,
-  PasswordReset_OUT
-}
-import com.softwaremill.bootzooka.test.{TestDependencies, BaseTest, Requests}
-import org.http4s._
-import org.http4s.syntax.all._
+import com.softwaremill.bootzooka.passwordreset.PasswordResetApi.{ForgotPassword_OUT, PasswordReset_OUT}
+import com.softwaremill.bootzooka.test._
 import org.scalatest.concurrent.Eventually
+import sttp.model.StatusCode
 
 class PasswordResetApiTest extends BaseTest with Eventually with TestDependencies {
-  lazy val requests = new Requests(dependencies.api)
-  import requests._
+  import testClient._
```

Comment:
maybe it would in fact be more readable if we removed this and simply wrote e.g. `requests.newRegisteredUser()` in the test. Should be clear what this method does